### ISSUE TITLE
[V3] artifact trigger doesn't kick off automatically

### ISSUE THREAD

**Issue by mefellows** [pact-foundation/pact-js#601]
Currently, our release process for the V3 package requires somebody to manually edit a release (e.g. by putting a blank space at the end of the release notes for it) to trigger it off. If left alone, you will have a release with no artifacts, rendering the release unusable:

![Screen Shot 2021-02-09 at 11 06 51 am](https://user-images.githubusercontent.com/53900/107297188-eac85b00-6ac6-11eb-9c67-9e016b4c91e7.png)

I believe it's due to this: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#triggering-new-workflows-using-a-personal-access-token

> When you use the repository's GITHUB_TOKEN to perform tasks on behalf of the GitHub Actions app, events triggered by the GITHUB_TOKEN will not create a new workflow run. This prevents you from accidentally creating recursive workflow runs. For example, if a workflow run pushes code using the repository's GITHUB_TOKEN, a new workflow will not run even when the repository contains a workflow configured to run when push events occur. For more information, see "Authenticating with the GITHUB_TOKEN."

So basically, when the v3 publish step completes, because it is using the GITHUB_TOKEN mechanism, it does not fire the

## Proposal

A tactical option would be to use a GH trigger action at the end of the workflow to trigger the artifact release step.

Butideally, the artifacts are published to the release before the npm package is published. We currently have a situation where there is a lag between "publish to npm" and the package being functional, which is not a big problem now, but will be a problem when it is the main

**Comment by mefellows** [comment]
This commit fixes the initial problem of triggering the artifact build: 9a6bcfe

**Comment by TimothyJones** [comment]
@mefellows : I think this is fixed now, is that correct?

**Comment by mefellows** [comment]
Yep!
